composer.json

```diff
@@ -38,7 +38,8 @@
         "phpcbf": "\"vendor/bin/phpcbf\"",
         "phpcbf-tests": "\"vendor/bin/phpcbf\" --standard=phpcs-test-ruleset.xml -s ./tests/",
         "sniffs": "\"vendor/bin/phpcs\" -e",
-        "test": "\"vendor/bin/phpunit\" --coverage-text",
+        "test": "\"vendor/bin/phpunit\"",
+        "test-cov": "\"vendor/bin/phpunit\" --coverage-text",
```

Comment:
Added this command to check coverage, default one does not (to speed up the checks).